sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Models/OperationsMockTests.cs

```diff
@@ -201,6 +201,41 @@ public void RecognizeInvoicesOperationRequiredParameters()
             Assert.Throws<ArgumentNullException>(() => new RecognizeInvoicesOperation("00000000 - 0000 - 0000 - 0000 - 000000000000", null));
         }
 
+        [Test]
+        public async Task RecognizeIdDocumentsOperationCreatesDiagnosticScopeOnUpdate()
```

Comment:
Remembered to update the scope name thanks to this test!

Reply:
I might be wrong, but I was seeing in translator that when the diagnostic scope was wrong, the recording will fail